examples/contracts/custom/src/multitest/tests.rs

```diff
@@ -24,4 +24,9 @@ fn test_custom() {
 
     let count = contract.query_custom().unwrap().count;
     assert_eq!(count, 1);
+
+    contract.sudo_custom().unwrap();
```

Comment:
Is there any test case asserting that when we call sudo_custom, the Sudo message is generated and sent under the hood? I can imagine a situation where `#[msg(sudo)]` and `sudo_custom()` might, by mistake, operate on an Exec or Query message.

Reply:
tracked under #316